### ISSUE TITLE
Batch Reimbursement

### ISSUE THREAD

**Issue by leo816** [bisq-network/support#1663]
Previous batch reimbursement #1623

**Comment by leo816** [comment]
The batch transaction with txid
[9599c694c459065be168b12e48c7229baabbb86b12d40152b50f9561abe31f0f](https://mempool.space/tx/9599c694c459065be168b12e48c7229baabbb86b12d40152b50f9561abe31f0f)


<img width="957" alt="Screen Shot 2024-08-05 at 12 57 17" src="https://github.com/user-attachments/assets/bdc3f40b-a1c2-470a-9bcb-b906c915a3ec">



has been broadcast and should be included in blocks soon.

The following process could not be carried out due to some incorrect payouts in the past. The payout was made manually by introducing each bitcoin address provided by each user and introducing the corresponding amount of fees:

The transaction was created against support-703.csv file at the root of this repo at commit b5414de. The csv file for creating electrum transaction was created with the following command:

`$ gawk -F, '{ addresses[$3] += $4+$5 } END { for (address in addresses) printf "%s,%f\n", address, addresses[address]}' support-515.csv |grep -v address > reimbursement.csv`

This prints out addresses to reimburse (column 3) and sums up the trading and mining fee per address (column 4 and 5). As there are payouts to the same address the script sums them up so there is only one line in the csv file, reducing the resulting transaction size and fee. The command also eliminates the header line.

The resulting csv file was fed to Electrum in the Send screen using the Tools -> Pay to many menu option.

**Comment by pazza83** [comment]
@leo816 there are 24 outputs for this reimbursement transaction. I assume one is the change address.

Therefore, 23 reimbursement issues were paid out.

On the report only 4 issues are linked. Are there more issues paid out other than those listed?

I also think it would be good to reference the total amount of bitcoin reimbursed in these reports otherwise one has to guess which is the change address, or use a process of elimination if all issues are correctly referenced.

**Comment by leo816** [comment]
Thank you for the comment @pazza83 , here is the response:

> @leo816 there are 24 outputs for this reimbursement transaction. I assume one is the change address.

Correct
> 
> Therefore, 23 reimbursement issues were paid out.

Correct, I think it was actually 20 issues because one of them contained several reimbursements within one.
> 
> On the report only 4 issues are linked. Are there more issues paid out other than those listed?

Yes, the report was incomplete and I have just added the rest.
> 
> I also think it would be good to reference the total amount of bitcoin reimbursed in these reports otherwise one has to guess which is the change address, or use a process of elimination if all issues are correctly referenced.

Yes, will add the specific amounts of btc on the role report.

Let me know if there is anything else

best,

**Comment by pazza83** [comment]
Thanks @leo816 it is a lot easier to follow now.

Was https://github.com/bisq-network/support/issues/1650 also reimbursed?

Specifiying the BSQ/BTC price used for reimursement on the report would also be useful for those with BSQ fees reimbursed.

**Comment by leo816** [comment]
No, that one was on the "waiting for info" column and I missed it. 

Just moved it to "ready" for next cycle. Thank you.

**Comment by pazza83** [comment]
Are you sure?

The address listed for payment bc1qpydyhrdmzpc42448tymgmckvxd4d0ajltd5ykj received 4 payouts.

What where the issues associate with these payouts?

I thought it would be:

https://github.com/bisq-network/support/issues/1648
https://github.com/bisq-network/support/issues/1649
https://github.com/bisq-network/support/issues/1650
https://github.com/bisq-network/support/issues/1651

I think that they are the only 4 issues created by that user?
